### PR TITLE
APPS-1466 print-nextflow-run-cmd

### PR DESCRIPTION
**Pull request by YuxinShi0423**
Screen Shot 2022-10-24 at 1 59 37 PM
https://staging.dnanexus.com/projects/GJ839q80QpbFg9BZ1Jk5J6Yg/monitor/job/GJFPX480QpbKZ511G7B8gzgK

### CODE REVIEW

**Comment by commandlinegirl** [comment]
Running integration tests on this branch (on test_nextflow module).

**Comment by commandlinegirl** [comment]
@YuxinShi0423 Is it possible to align the ":" characters in each line? I just rerun the quoted job on STG and got the same output as above:
image

But perhaps that job uses an older version of this PR?

File: src/python/dxpy/templating/templates/nextflow/src/nextflow.sh [hunk 1]
```diff
@@ -161,7 +156,18 @@ main() {
...
-      @@REQUIRED_RUNTIME_PARAMS@@ & NXF_EXEC_PID=$!
+      @@REQUIRED_RUNTIME_PARAMS@@
+      "
+
+    trap on_exit EXIT
+    echo "============================================================="
+    echo "=== NF work-dir : ${DX_WORK}"
+    echo "=== NF log file : ${DX_LOG}"
+    echo "=== NF cache    : $DX_PROJECT_CONTEXT_ID:/.nextflow/cache/$NXF_UUID"
+    echo "=== NF command  :" $NEXTFLOW_CMD
```
Comment (YuxinShi0423) [inline src/python/dxpy/templating/templates/nextflow/src/nextflow.sh:167]: Fixed already. Sorry my snapshot is not updated. @commandlinegirl

**Review by commandlinegirl — APPROVED** [review]
LGTM, thank you!

**Pull request merged**